src/com/epam/javatrainings/threads/timer/Main.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.javatrainings.threads.timer;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.print("Enter minutes: ");
+        int minute = input.nextInt();
+        System.out.print("Enter seconds: ");
+        int second = input.nextInt();
```

Comment:
Need to validate the minutes and seconds before executing.
For min < 0 && min > 59 && sec < 0 && sec > 59

Reply:
done!